script/test/fixtures/v3/docker-compose-deploy-mode.yaml

```diff
@@ -0,0 +1,9 @@
+version: "3"
+
+services:
+  foo:
+    deploy:
+      resources:
```

Comment:
`mode: ` is not under resource but directly under `deploy` key and the same thing for `replicas`